test/Macros/accessor_macros.swift

```diff
@@ -17,7 +17,7 @@
 // REQUIRES: OS=macosx
 
 @attached(accessor)
-macro myPropertyWrapper: Void =
+macro myPropertyWrapper() =
```

Comment:
I noted this offline, but we should consider this change in the context of expression macros as well. Perhaps macros should always have argument lists, but can be used without parentheses if all parameters have default arguments.